### PR TITLE
tags/instructor/*.tag: change to 2-space indentation #7352

### PR DESCRIPTION
**Pull request by sujeet14108**
Fixes partially #7352
All files in `tags/instructor/search/*.tag` and `tags/instructor/feedbackEdit/*.tag`

### CODE REVIEW

**Comment by sujeet14108** [comment]
Ready for review :)

File: src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/newQuestionForm.tag [hunk 1]
```diff
@@ -11,134 +11,134 @@
 <c:set var="NEW_QUESTION" value="-1" />
 
 <form id="form_editquestion-${NEW_QUESTION}" class="form-horizontal form_question tally-checkboxes" role="form" method="post"
-    action="<%= Const.ActionURIs.INSTRUCTOR_FEEDBACK_QUESTION_ADD %>"
-    name="form_addquestions" data-qnnumber="${NEW_QUESTION}">
-    <div class="well well-plain inputTable" id="addNewQuestionTable">
+  action="<%= Const.ActionURIs.INSTRUCTOR_FEEDBACK_QUESTION_ADD %>"
+  name="form_addquestions" data-qnnumber="${NEW_QUESTION}">
```
Comment (leeyimin) [inline src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/newQuestionForm.tag]: 2 spaces more.

File: src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/newQuestionForm.tag [hunk 2]
```diff
@@ -11,134 +11,134 @@
...
+              type="button" data-toggle="dropdown">
+            Add New Question <span class="caret"></span>
+          </button>
+          <ul id="add-new-question-dropdown" class="dropdown-menu">
+            ${fqForm.questionTypeOptions}
+          </ul>
         </div>
+
+        <a href="/instructorHelp.jsp#fbQuestionTypes"
+          target="_blank" rel="noopener noreferrer">
```
Comment (leeyimin) [inline src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/newQuestionForm.tag]: 2 spaces more.

File: src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/newQuestionForm.tag [hunk 3]
```diff
@@ -11,134 +11,134 @@
...
-            </div>
+  <div class="panel panel-primary questionTable" id="questionTable-${NEW_QUESTION}" style="display:none;">
+    <div class="panel-heading">
+      <div class="row">
+        <div class="col-sm-7">
+          <span>
+            <strong>Question</strong>
+            <select class="questionNumber nonDestructive text-primary"
+              name="<%= Const.ParamsNames.FEEDBACK_QUESTION_NUMBER %>"
+              id="<%= Const.ParamsNames.FEEDBACK_QUESTION_NUMBER %>">
```
Comment (leeyimin) [inline src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/newQuestionForm.tag]: 2 spaces more.

File: src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/newQuestionForm.tag [hunk 4]
```diff
@@ -11,134 +11,134 @@
...
-                <span class="pull-right">
-                    <button id="button_submit_add" class="btn btn-primary" type="submit" tabindex="9">
-                        Save Question
-                    </button>
-                </span>
+        <div class="col-sm-5 mobile-margin-top-10px">
+          <span class="mobile-no-pull pull-right">
+            <a class="btn btn-primary btn-xs btn-discard-changes"
+              data-qnnumber="${NEW_QUESTION}" data-toggle="tooltip" data-placement="top"
+              title="<%= Const.Tooltips.FEEDBACK_QUESTION_CANCEL_NEW %>">
```
Comment (leeyimin) [inline src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/newQuestionForm.tag]: 2 spaces more.

File: src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/newQuestionForm.tag [hunk 5]
```diff
@@ -11,134 +11,134 @@
...
+          <div class="col-sm-10">
+            <%-- Do not add whitespace between the opening and closing tags --%>
+            <textarea class="form-control textvalue nonDestructive" rows="2"
+              name="<%= Const.ParamsNames.FEEDBACK_QUESTION_TEXT %>"
+              id="<%= Const.ParamsNames.FEEDBACK_QUESTION_TEXT %>-${NEW_QUESTION}"
+              data-toggle="tooltip" data-placement="top"
+              title="<%= Const.Tooltips.FEEDBACK_QUESTION_INPUT_INSTRUCTIONS %>"
+              placeholder="<%= Const.PlaceholderText.FEEDBACK_QUESTION %>"
+              tabindex="9"
+              disabled></textarea>
```
Comment (leeyimin) [inline src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/newQuestionForm.tag]: 2 spaces more.

File: src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/newQuestionForm.tag [hunk 6]
```diff
@@ -11,134 +11,134 @@
...
+              [Optional]<br>Description
+            </label>
+          </h5>
+          <div class="col-sm-10">
+            <div class="panel panel-default panel-body question-description"
+              id="<%= Const.ParamsNames.FEEDBACK_QUESTION_DESCRIPTION %>-${NEW_QUESTION}"
+              data-toggle="tooltip" data-placement="top"
+              title="<%= Const.Tooltips.FEEDBACK_QUESTION_INPUT_DESCRIPTION %>"
+              data-placeholder="<%= Const.PlaceholderText.FEEDBACK_QUESTION_DESCRIPTION %>"
+              tabindex="9">
```
Comment (leeyimin) [inline src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/newQuestionForm.tag]: 2 spaces more.

File: src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/newQuestionForm.tag [hunk 7]
```diff
@@ -11,134 +11,134 @@
...
+  <input type="hidden" id="<%= Const.ParamsNames.FEEDBACK_QUESTION_TYPE %>" name="<%= Const.ParamsNames.FEEDBACK_QUESTION_TYPE %>">
+  <input type="hidden" name="<%= Const.ParamsNames.FEEDBACK_SESSION_NAME %>" value="${fqForm.feedbackSessionName}">
+  <input type="hidden" name="<%= Const.ParamsNames.COURSE_ID %>" value="${fqForm.courseId}">
+  <input type="hidden" name="<%= Const.ParamsNames.FEEDBACK_QUESTION_SHOWRESPONSESTO %>" >
+  <input type="hidden" name="<%= Const.ParamsNames.FEEDBACK_QUESTION_SHOWGIVERTO %>" >
+  <input type="hidden" name="<%= Const.ParamsNames.FEEDBACK_QUESTION_SHOWRECIPIENTTO %>" >
+  <input type="hidden" name="<%= Const.ParamsNames.USER_ID %>" value="${data.account.googleId}">
+  <input type="hidden" name="<%= Const.ParamsNames.FEEDBACK_QUESTION_GENERATEDOPTIONS %>"
+    value="<%= FeedbackParticipantType.NONE.toString() %>"
+    id="<%= Const.ParamsNames.FEEDBACK_QUESTION_GENERATEDOPTIONS %>">
```
Comment (leeyimin) [inline src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/newQuestionForm.tag]: 2 spaces more.

File: src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/previewSessionForm.tag [hunk 8]
```diff
@@ -6,55 +6,55 @@
...
-                </div>
-                <input type="hidden" name="<%= Const.ParamsNames.USER_ID %>" value="${data.account.googleId}">
-            </form>
+        <input type="hidden" name="<%= Const.ParamsNames.FEEDBACK_SESSION_NAME %>" value="${previewForm.fsName}">
+        <input type="hidden" name="<%= Const.ParamsNames.COURSE_ID %>" value="${previewForm.courseId}">
+        <div class="col-sm-6">
+          <input id="button_preview_student" type="submit" class="btn btn-primary" value="Preview as Student"
+              <c:if test="${empty previewForm.studentToPreviewAsOptions}"> disabled style="background: #66727A;"
+              </c:if>
+          >
```
Comment (leeyimin) [inline src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/previewSessionForm.tag]: Indentation of lines 31 to 34 can be improved.
One possible way of indenting these lines:
```
<input id="button_preview_student" type="submit" class="btn btn-primary" value="Preview as Student"
    <c:if test="${empty previewForm.studentToPreviewAsOptions}"> disabled style="background: #66727A;"</c:if>>
```

File: src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/questionEditForm.tag [hunk 9]
```diff
@@ -14,118 +14,118 @@
...
-                            ${fqForm.questionDescription}
-                        </div>
-                        <input type="hidden" name="<%= Const.ParamsNames.FEEDBACK_QUESTION_DESCRIPTION %>">
-                    </div>
-                    ${fqForm.questionSpecificEditFormHtml}
-                </div>
+        <div class="form-group" style="padding: 0 15px;">
+          <h5 class="col-sm-2">
+            <label class="align-left"
+              for="<%= Const.ParamsNames.FEEDBACK_QUESTION_DESCRIPTION %>-${fqForm.questionIndex}">
```
Comment (leeyimin) [inline src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/questionEditForm.tag]: 2 spaces more.

File: src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/questionFeedbackPathSettings.tag [hunk 10]
```diff
@@ -8,108 +8,108 @@
...
+            id="<%= Const.ParamsNames.FEEDBACK_QUESTION_RECIPIENTTYPE %>-${fqForm.questionIndex}"
+            name="<%= Const.ParamsNames.FEEDBACK_QUESTION_RECIPIENTTYPE %>"
+            <c:if test="${!fqForm.editable}">disabled</c:if>>
+          <c:forEach items="<%= FeedbackParticipantType.RECIPIENTS %>" var="recipientType">
+            <option <c:if test="${fqForm.feedbackPathSettings.selectedRecipient eq recipientType}">selected </c:if>value="${recipientType}">
+              ${recipientType.displayNameRecipient}
+            </option>
+          </c:forEach>
+        </select>
     </div>
```
Comment (leeyimin) [inline src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/questionFeedbackPathSettings.tag]: 2 spaces more. Indentation below is also wrong.

File: src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/questionVisibilityOptions.tag [hunk 11]
```diff
@@ -13,155 +13,155 @@
...
+              <c:if test="${fqForm.visibilitySettings.recipientNameVisibleFor[FEEDBACK_INSTRUCTORS]}"> checked</c:if> >
+        </td>
+      </tr>
+    </table>
+  </div>
+  <!-- Fix for collapsing margin problem. Reference: http://stackoverflow.com/questions/6204670 -->
+  <div class="visibilityMessage overflow-hidden" id="visibilityMessage-${fqForm.questionIndex}">
+    This is the visibility hint as seen by the feedback giver:
+    <ul class="text-muted background-color-warning">
+    <c:forEach items="${fqForm.visibilitySettings.visibilityMessages}" var="msg">
```
Comment (leeyimin) [inline src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/questionVisibilityOptions.tag]: 2 spaces more.

File: src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/newQuestionForm.tag [hunk 12]
```diff
@@ -11,134 +11,134 @@
...
+              title="<%= Const.Tooltips.FEEDBACK_QUESTION_INPUT_INSTRUCTIONS %>"
+              placeholder="<%= Const.PlaceholderText.FEEDBACK_QUESTION %>"
+              tabindex="9"
+              disabled></textarea>
+          </div>
+        </div>
+        <div class="form-group" style="padding: 0 15px;">
+          <h5 class="col-sm-2">
+            <label class="align-left"
+              for="<%= Const.ParamsNames.FEEDBACK_QUESTION_DESCRIPTION %>-${NEW_QUESTION}">
```
Comment (leeyimin) [inline src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/newQuestionForm.tag]: 2 spaces more.

Comment (sujeet14108) [inline src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/newQuestionForm.tag]: Done

File: src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/questionFeedbackPathSettings.tag [hunk 13]
```diff
@@ -8,108 +8,108 @@
...
-                    name="<%= Const.ParamsNames.FEEDBACK_QUESTION_GIVERTYPE %>"
-                    <c:if test="${!fqForm.editable}">disabled</c:if>>
-                    <c:forEach items="<%= FeedbackParticipantType.GIVERS %>" var="giverType">
-                        <option <c:if test="${fqForm.feedbackPathSettings.selectedGiver eq giverType}">selected </c:if>value="${giverType}">
-                            ${giverType.displayNameGiver}
-                        </option>
-                    </c:forEach>
-                </select>
-            </div>
+  <div class="col-sm-12 row numberOfEntitiesElements">
```
Comment (leeyimin) [inline src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/questionFeedbackPathSettings.tag]: Indentation here is wrong.

**Pull request merged**